gstorm/include/detail/ranges/vector.h

```diff
@@ -355,7 +367,9 @@ struct gvector : public traits::range_forward_traits<T>, public range::gvector_b
   // this is called by the gvector's base class gvector_base and is triggert after the
   // executor updates the gvectors cgh
   virtual void updateAccessors() override {
```

Comment:
Eliminate `virtual`

Reply:
Called using a pointer to the base class.

Reply:
I agree, virtual is a little bit verbose here. Override indicates the virtual nature of the function

Reply:
probably can be final too ?

Reply:
True